docs/solidity-to-cadence.md

```diff
@@ -133,50 +136,54 @@ authorized it. Doing so is key to access and security, and is the basis of ident
+     width="245" align="left" alt="In Cadence, the subject must possess the Capability to access the protected
+     resource"/>
 
-The [capability-based security](https://en.wikipedia.org/wiki/Capability-based_security) model frames access in
-the opposite direction than the [access-based security](https://en.wikipedia.org/wiki/Access-control_list) model.
+The [capability-based security](https://en.wikipedia.org/wiki/Capability-based_security) model frames access in the
+opposite direction than the [access-based security](https://en.wikipedia.org/wiki/Access-control_list) model.
 
 
 ## Access control using Capabilities
 
-Solidity lacks specific types or other primitives to aid with permission management.
-Developers have to inline guards to `require` at every function entry-point,
-thus validating the `msg.sender` of the transaction.
+Solidity lacks specific types or other primitives to aid with permission management. Developers have to inline guards to
+`require` at every function entry-point, thus validating the `msg.sender` of the transaction.
 
-[Capabilities](./language/capabilities.md) are defined by linking storage paths (namespaces for contract
-storage) to protected objects and then making that linked capability available to other accounts.
+[Capabilities](./language/capabilities.md) are defined by linking storage paths (namespaces for contract storage) to
+protected objects and then making that linked capability available to other accounts.
 
-Any account can get access to an account's public Capabilities. Public capabilities are created using public paths,
-i.e. they have the domain `public`. For example, all accounts have a default public capability linked to the
-`FlowToken.Vault` Resource. This vault is exposed as a public capability, scoped to the `FungibleToken.Receiver` resource
-interface, to allow any account to `borrow()` a reference to the Vault to make a `deposit()`. Since only the functions
-defined under the `[FungibleToken.Receiver](https://github.com/onflow/flow-ft/blob/master/contracts/FungibleToken.cdc#L105)`
-interface are exposed, the borrower of the vault reference cannot call `withdraw()` since it is scoped in the `provider`
-interface.
+Any account can get access to an account's public Capabilities. Public capabilities are created using public paths, i.e.
+they have the domain `public`. For example, all accounts have a default public capability linked to the
+`FlowToken.Vault` Resource. This vault is exposed as a public unentitled capability allowing any account to `borrow()` a
+reference to the Vault to make a `deposit()`. Since only the unentitled functions defined under the
+[`FungibleToken.Vault`](https://github.com/onflow/flow-ft/blob/master/contracts/FungibleToken.cdc#L167) interface are
+exposed, the borrower of the vault reference cannot call `withdraw()` since the method requires a `Withdraw` entitled
```

Comment:
```suggestion
interface are exposed, the borrower of the vault reference cannot call `withdraw()` since the method requires a `Withdraw` entitled
```